scripts/ci/docker-compose/backend-mysql.yml

```diff
@@ -25,6 +25,8 @@ services:
       - AIRFLOW__CELERY__RESULT_BACKEND=db+mysql://root@mysql/airflow?charset=utf8mb4
     depends_on:
       - mysql
+    ports:
```

Comment:
Currently we are using it local.yml for CI build. But it's not used in PROD image. I think we should have another "local-prod.yml" and expose PORT there. then you'd need to update `prepare_command_files()` function in `breeze` and modify COMPOSE_PROD_FILE to add this local-prod.yml to this variable inside this if:.
```
    if [[ "${MOUNT_LOCAL_SOURCES}" != "false" ]]; then
        COMPOSE_CI_FILE=${COMPOSE_CI_FILE}:${LOCAL_DOCKER_COMPOSE_FILE}
    fi
```

Reply:
Like this https://github.com/apache/airflow/pull/8228/files ?